module/VuFind/src/VuFind/Auth/Shibboleth/Shibboleth.php

```diff
@@ -27,15 +27,18 @@
  * @author   Bernd Oberknapp <bo@ub.uni-freiburg.de>
  * @author   Demian Katz <demian.katz@villanova.edu>
  * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @author   Vaclav Rosecky <vaclav.rosecky@mzk.cz>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
-namespace VuFind\Auth;
+namespace VuFind\Auth\Shibboleth;
```

Comment:
Is it necessary to move everything into the Shibboleth namespace? If we keep the file where it was but add support files within the Shibboleth namespace, I think that would be equally valid, but slightly less disruptive.